packages/entity-server/jest.setup.ts

```diff
@@ -4,6 +4,11 @@
  */
 
 import { getTestDatabase, clearTestData } from '@tupaia/database';
+import { setupTestData } from './src/__tests__/testUtilities';
+
+beforeAll(async () => {
+  await setupTestData();
+});
```

Comment:
Just do this once at the start of all the tests rather than once per test. Initially I had assumed we might want to insert different data per test, but in practice it had proved easier and cleaner to just import the same data each time. So in the end it's faster to just import it once at the start of all tests.